frontend/.eslintrc.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true
+  },
+  extends: [
+    "plugin:react/recommended",
+    "standard"
+  ],
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true
+    },
+    ecmaVersion: 12,
+    sourceType: "module"
+  },
+  plugins: [
+    "react",
+    "@typescript-eslint"
+  ],
+  rules: {
+    quotes: ["error", "double"],
+    // Keeping no-unused-vars on had some weird behavior with enums
+    semi: ["error", "always"],
+    "no-unused-vars": "off",
```

Comment:
Unused variables is imo one of the most useful lints out there. What issues with enums did we see? I think we have some enums which we never construct / read one value of, which actually *does* indicate a problem. Can we consider fixing, `// eslint-disable-next-line`-ing them, or at the very least, turning these to warnings?